application/common/models/User.php

```diff
@@ -580,26 +580,35 @@ public function setPassword($newPassword)
     }
 
     /**
-     * @param string $clientId
+     * @param string $authType
      * @return string
      * @throws \Exception
      */
-    public function createAccessToken($clientId, $authType)
+    public function createAccessToken($authType)
     {
         /*
          * Create access_token and update user
          */
         $accessToken = Utils::generateRandomString(32);
         /*
-         * Store combination of clientId and accessToken for bearer auth
+         * Store accessToken for auth
          */
         $this->auth_type = $authType;
-        $this->access_token = Utils::getAccessTokenHash($clientId . $accessToken);
+        $this->access_token = Utils::getAccessTokenHash($accessToken);
         $this->access_token_expiration = Utils::getDatetime(
             time() + \Yii::$app->params['accessTokenLifetime']
         );
         $this->saveOrError('Unable to create access token', 1465833228);
 
+        $secure = (YII_ENV != 'dev' || YII_ENV != 'test') ? true : false;
```

Comment:
This is always true. Also, ternary does nothing. Possibly consider using !in_array() as well.

Reply:
maybe that's why I was having cors issues locally

Reply:
nice catch btw.